app/transport/http/resp.go

```diff
@@ -0,0 +1,25 @@
+package http
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+)
+
+func GenerateJSONResponse(w http.ResponseWriter, statusCode int, data interface{}) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(statusCode)
+
+	if data == nil {
+		log.Println("Data can not be empty")
+		return
+	}
+
+	if err := json.NewEncoder(w).Encode(data); err != nil {
+		fmt.Printf("could not encode json: %v", err)
```

Comment:
i don`t think we need to write some text  into console at all

Reply:
Actually, we have to write some kind of message using logger (one of our tasks for current sprint) to our backend side and of course status codes and messages for the client.

Reply:
> Actually, we have to write some kind of message using logger (one of our tasks for the current sprint) to our backend side and of course status codes and messages for the client. Let's focus on the second part for now

Yes. Using Log not fmt. In line 15 log is using, in line 20 fmt )